src/cart/crt_swim.c

```diff
@@ -144,32 +200,38 @@ static int crt_swim_send_message(struct swim_context *ctx, swim_id_t to,
 	ep.ep_tag  = csm->csm_crt_ctx_idx;
 	D_RWLOCK_UNLOCK(&csm->csm_rwlock);
 
-	/* get the opcode of the first RPC in version 0 of OPC_SWIM_PROTO */
-	opc = CRT_PROTO_OPC(CRT_OPC_SWIM_PROTO, CRT_OPC_SWIM_VERSION, 0);
+	if (nupds > 0 && upds[0].smu_state.sms_status == SWIM_MEMBER_INACTIVE)
+		opc_idx = 1;
+
+	opc = CRT_PROTO_OPC(CRT_OPC_SWIM_PROTO, CRT_OPC_SWIM_VERSION, opc_idx);
 	rc = crt_req_create(crt_ctx, &ep, opc, &rpc_req);
 	if (rc) {
 		D_ERROR("crt_req_create() failed rc=%d\n", rc);
 		D_GOTO(out, rc);
 	}
 
-	rc = crt_req_set_timeout(rpc_req, CRT_SWIM_RPC_TIMEOUT);
-	if (rc) {
-		D_ERROR("crt_req_set_timeout() failed rc=%d\n", rc);
-		crt_req_decref(rpc_req);
-		D_GOTO(out, rc);
+	if (opc_idx == 0) { /* set timeout for one way RPC only */
+		rc = crt_req_set_timeout(rpc_req, CRT_SWIM_RPC_TIMEOUT);
+		if (rc) {
+			D_TRACE_ERROR(rpc_req,
```

Comment:
RPC_ERROR?

Reply:
the same as above related to usage RPC_* macros.